pkg/storage/iterator.go

```diff
@@ -170,7 +171,7 @@ func (it *batchChunkIterator) Next() bool {
 func (it *batchChunkIterator) nextBatch() (iter.EntryIterator, error) {
 	// the first chunk of the batch
 	headChunk := it.chunks.Peek()
-
+	it.storeStats.TotalChunksOverlapping += int64(len(it.lastOverlapping))
```

Comment:
I was wondering if this could be moved inside the block with the condition `if it.chunks.Len() > 0 `, after `it.lastOverlapping` list is built?

Reply:
yes it's more or less the same. I can make that change though.